components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -94,6 +97,26 @@ function InputBox(props: InputBoxProps) {
     }
   }
 
+  const updateAttachmentWrapperHeight = () => {
+    let { height } = attachmentWrapperRef?.current?.getBoundingClientRect() ?? {
+      height: 0
+    }
+    setAttachmentWrapperHeight(height)
+  }
+
+  React.useEffect(() => {
+    // Set the height of the attachment wrapper on initial render
+    setTimeout(() => {
+      updateAttachmentWrapperHeight()
+    }, 0)
```

Comment:
note: this isn't only going to happen on the initial render - whenever `pendingMessageImages` changes this will run

i.e. it'll run twice on image changes

Reply:
Removed this `setTimeout` https://github.com/brave/brave-core/compare/ae9529f277e1e45312894c622b91ba54974079b7..cdad76d44dc9f36f8506afe080354347812315f4